### ISSUE TITLE
Limbo is too dark

### ISSUE THREAD

**Issue by ScytheIronclaw** [DimensionalDevelopment/DimDoors#40]
After exploring through some dungeon rifts and geting to W 92, I fell into a trap. After entering Limbo I know how to normally escape, go to the lowest point and find Eternal Fabric.

- [x] After finding Eternal Fabric and jumping on it nothing happened.
I haven't seen any other dungeon rifts in limbo to escape into. The only other option in non-cheat survival is to dig out of the bottom of the world and die.

- [ ] Also it's very dark and Limbo and makes navigating around the bottom of the dimension difficult. Perhaps make the unraveled fabric a bit brighter?

**Comment by ZombieHDGaming** [comment]
This is a known issue with Eternal Fabric, this will be fixed in the future when we switch it to a liquid. As for the darkness of Limbo, we have done nothing between the old versions and this, and have heard nothing from other players. What is your brightness setting?

**Comment by Runemoro** [comment]
Yes, eternal fabric needs to be fixed. The only way to escape limbo is to dig down right now.

**Comment by Robijnvogel** [comment]
@ScytheIronclaw This is partially a duplicate of #31 
"Repair Eternal Fabric's ability to teleport the player back to the Overworld"

**Comment by Runemoro** [comment]
Limbo isn't too dark. It's meant to be like that to confuse players. I'd actually say that since changing to the new texture it's too bright.

So this is partially duplicate and partially works as intended.

**Comment by ScytheIronclaw** [comment]
Okay, nice to know that this is intentional. At least you can modify the textures via Resource Packs. So this is a  non-issue.

**Comment by scribbler957** [comment]
Not to necro, but dark/light depends a lot on the monitor setting and system. I've got my monitor well calibrated so black is really black, not shiny gray, and when I go to limbo I just dig into the void because I can't see anything at all.

**Comment by Robijnvogel** [comment]
I'm going to re-open this issue following because of [this](https://minecraft.curseforge.com/projects/dimensionaldoors?gameCategorySlug=mc-mods&projectID=284876#c134) 

We will probably need to add a config option to make Limbo slightly lighter for players who experience Limbo as way too dark.
